templates/default/layouts/docs.js

```diff
@@ -1,12 +1,41 @@
+import siteConfig from '../config/siteConfig';
+
 export default function DocsLayout({ children, frontMatter }) {
-  const { title } = frontMatter;
+  const { title, editLink } = frontMatter;
+  // edit link will only appear if enabled in frontmatter and in siteConfig
+  const editUrl = siteConfig.repoRoot + siteConfig.repoEditPath + frontMatter._raw.sourceFilePath;
   return (
     <article className="docs prose dark:prose-invert prose-a:break-words mx-auto p-6">
       <header>
         <div className="mb-6">{title && <h1>{title}</h1>}</div>
       </header>
       <section>
         {children}
+        {editUrl && siteConfig.editPageShow && editLink && (
```

Comment:
...so, in this case if I switch `editPageShow` to `true` but in `configlayer.config` we have `editLink` still set to false, the button will not show up.